src/CodeGen/ODataT4CodeGenerator.cs

```diff
@@ -6815,8 +6815,8 @@ internal override void WriteCastToMethods(string baseTypeName, string derivedTyp
 this.Write(this.ToStringHelper.ToStringWithCulture(derivedTypeFullName));
 
 this.Write("\r\n        \'\'\' </summary>\r\n        \'\'\' <param name=\"source\">source entity</param>\r" +
-        "\n        <Global.System.Runtime.CompilerServices.Extension()>\r\n        Public Fu" +
-        "nction CastTo");
+        "\n        <Global.System.Runtime.CompilerServices.Extension()>\r\n        Public " +
+        "Function CastTo");

```

Comment:
Sorry, what's the difference?

Reply:
no difference, just PoliCheck doesn't like the apparent word "Fu".